### PR TITLE
Use Nette\SmartObject to fix PHP 7.2 compability

### PR DESCRIPTION
**Pull request by Salamek**
Compability with PHP 7.2, sadly this means to drop nette 2.3 compability :cry:

### CODE REVIEW

File: src/IPub/Gravatar/Gravatar.php [hunk 1]
```diff
@@ -33,8 +33,9 @@
  *
  * @author         Adam Kadlec <adam.kadlec@ipublikuj.eu>
  */
-final class Gravatar extends Nette\Object
+final class Gravatar
 {
+	use Nette\SmartObject;
```
Comment (akadlec) [inline src/IPub/Gravatar/Gravatar.php]: pls one blank line.

**Comment by Salamek** [comment]
@akadlec looks like CI will fail, network error on travis side... so you may need to rerun CI...

**Comment by Salamek** [comment]
@akadlec merge && release pls ?

**Comment by Salamek** [comment]
THX!

**Pull request merged**